console-plugins/map-optimization-plugin/src/optimizer-plugin.cc

```diff
@@ -30,19 +27,29 @@ OptimizerPlugin::OptimizerPlugin(
   addCommand(
       {"noptimize_visual", "optv"},
```

Comment:
what does the n stand for in noptimize_visual?

Reply:
no clue, probably an old typo